pom.xml

```diff
@@ -28,6 +28,7 @@
 
   <properties>
     <tycho.scmUrl>scm:git:https://github.com/eclipse-platform/eclipse.platform.swt.git</tycho.scmUrl>
+    <tycho.version>4.0.13-SNAPSHOT</tycho.version>
```

Comment:
This should be removed before submitting this version also has to be used in I-builds then.

Reply:
Just saw that we are still using older pomless extensions throughout the projects (such as 4.0.8 in platform and 4.0.10 here):
https://github.com/eclipse-platform/eclipse.platform.swt/blob/cac15c334b8d2cfe71bd2fd55e0a53935af32b6b/.mvn/extensions.xml#L7

Probably not an actual issue though.